sanity.config.ts

```diff
@@ -40,6 +46,14 @@ export default defineConfig({
   // edit schemas in './src/schemas'
   schema: {
     types: schemaTypes,
+    templates: (templates) =>
+      templates.filter(({ schemaType }) => !singletonTypes.has(schemaType)),
+  },
```

Comment:
I found this in Sanity documentation but I don't know why we need that change to templates.
It works even without this change.